amplify/backend/api/rms/parameters.json

```diff
@@ -0,0 +1,5 @@
+{
+  "AppSyncApiName": "rms",
+  "DynamoDBBillingMode": "PAY_PER_REQUEST",
+  "DynamoDBEnableServerSideEncryption": false
+}
```

Comment:
Can you change the parameters to be this:

```
{
  "AppSyncApiName": "rms",
  "DynamoDBBillingMode": "PROVISIONED",
  "DynamoDBModelTableReadIOPS": 1,
  "DynamoDBModelTableWriteIOPS": 1,
  "DynamoDBEnablePointInTimeRecovery": false,
  "DynamoDBEnableServerSideEncryption": false
} 
```

Reply:
Changes made @yauj